__tests__/features/end-to-end.steps.ts

```diff
@@ -557,6 +559,19 @@ defineFeature(loadFeature("./end-to-end.feature"), test => {
         .sendKeys("Health notes");
       await browser!.submit();
 
+      // Disability page
+      await expect(browser!.getCurrentUrl()).resolves.toContain(
+        "/disability?process_type=thc"
+      );
+      await browser!.findElement({ id: "disability-yes" }).click();
+      await browser!.findElement({ id: "who-disability-tenant-1" }).click();
+      await browser!.findElement({ id: "pip-or-dla-yes" }).click();
+      await browser!.findElement({ id: "who-pip-tenant-1" }).click();
```

Comment:
Think this should be ` browser!.waitForEnabledElement({ id: id: "disability-yes" }).click();` etc